static/app/views/performance/landing/widgets/transforms/transformMetricsToArea.tsx

```diff
@@ -8,6 +7,11 @@ import {aggregateOutputType} from 'sentry/utils/discover/fields';
 import {MetricsRequestRenderProps} from 'sentry/utils/metrics/metricsRequest';
 
 import {QueryDefinitionWithKey, WidgetDataConstraint, WidgetPropUnion} from '../types';
+import {PerformanceWidgetSetting} from '../widgetDefinitions';
+
+// Sentry treats transactions with a status other than “ok,” “canceled,” and “unknown” as failures.
+// For more details, see https://docs.sentry.io/product/performance/metrics/#failure-rate
+const TRANSACTION_SUCCESS_STATUS = ['ok', 'unknown', 'canceled'];
```

Comment:
@priscilawebdev I think all instances of `canceled` should be renamed to `cancelled` since this is what it is filtered on https://github.com/getsentry/snuba/blob/881b4e43c288c62fc1d8bab28cc19c187ba525ec/snuba/query/processors/performance_expressions.py#L33 .


The documentation in https://docs.sentry.io/product/performance/metrics/#failure-rate has a typo (`canceled` should be `cancelled`).

Reply:
Yes, sorry about the confusion, it was `cancelled` initially but https://docs.sentry.io/product/performance/metrics/#failure-rate made be think it had to be changed.